README.md

```diff
@@ -224,18 +224,23 @@ The extension must be added to a parameter for consumption. This could be a head
 ```yaml
 parameters:
   bookId:
-      name: bookId
+      name: accountId
```

Comment:
Why change the variable name?

I don't mind, but you didn't change it on line 226. I'm just curious why you're changing it.

Reply:
I missed 226 😅
As for why I changed it to an account scenario instead of a book, at the time I thought it made more sense. In retrospect, book works just as well. I'll make a few changes and push it up. Would you mind adding me as a contributor for my previous work?